spread.yaml

```diff
@@ -8,15 +8,25 @@
 project: spread
 
 environment:
-    GOVERSION: 1.7.3
-    GOPATH: /home/test
-    PATH: $GOPATH/bin:$PATH
+    GOVERSION: 1.10.4
+    GOHOME: /home/test
+    GOPATH: $GOHOME
+    PATH: $GOHOME/bin:$PATH
     CACHE_DISABLED: $(HOST:[ "$SEND_CACHE" = 1 ] && echo false || echo tests/cache)
 
 backends:
+    google:
+        key: "$(HOST: echo $SPREAD_GOOGLE_KEY)"
+        location: computeengine/us-east1-b
+        halt-timeout: 2h
+        systems:
+            - ubuntu-18.04-64:
+                image: ubuntu-1804-64-virt-enabled
+                workers: 1
+
     qemu:
```

Comment:
Is this backend still working?

Reply:
It is not working for me on bionic with qemu 2.11.1, bu works using qemu on xenial.

Reply:
Missing response here.

Reply:
sorry, qemu still works